benchmarks/perf-tool/sample-configs/faiss-nsg/index-spec.json

```diff
@@ -0,0 +1,26 @@
+{
+  "settings": {
+    "index": {
+      "knn": true,
+      "refresh_interval": "10s",
+      "number_of_shards": 3,
+      "number_of_replicas": 0
+    }
+  },
+  "mappings": {
+    "properties": {
+      "target_field": {
+        "type": "knn_vector",
+        "dimension": 128,
+        "method": {
+          "name": "nsg",
+          "space_type": "l2",
+          "engine": "faiss",
+          "parameters": {
+            "m": 32
```

Comment:
do we have "m" parameter in NSG, I thought it was "R"

Reply:
at latest https://github.com/opensearch-project/k-NN/pull/945/commits/04333c8df5c357897e884c5afd495877b9ca8d2f i fixed with 'r'